niffler-e-2-e-tests/src/test/java/guru/qa/niffler/page/MainPage.java

```diff
@@ -17,4 +23,23 @@ public EditSpendingPage editSpending(String spendingDescription) {
   public void checkThatTableContainsSpending(String spendingDescription) {
     tableRows.find(text(spendingDescription)).should(visible);
   }
+
+  public ProfilePage clickToProfile(){
+
+    menuButton.click();
+    profileOption.click();
+
+    return new ProfilePage();
+  }
+
+  public MainPage headersBlocksShouldBeExists(){
+
+    String statisticsHeader = "Statistics";
+    String spendingHeader = "History of Spendings";
+
+    statisticBlock.shouldBe(exist).shouldHave(text(statisticsHeader));
+    spendingBlock.shouldBe(exist).shouldHave(text(spendingHeader));
+
```

Comment:
Я бы использовала здесь visible вместо exist.
Из документации селенида:
exist  -- Check if element exist. It can be visible or hidden.
visible -- Checks if element is visible


Ну и лишние пустые строки можно убрать. Дальше по коду тоже встречаются, не буду дублировать комментарий.